apps/music/test/marionette/Player_test.js

```diff
@@ -133,4 +133,39 @@ marionette('Music player tests', function() {
       music.checkPlayerIconShown(true);
     });
   });
+
+  suite('Rating test', function() {
+    test('Rating persistance', function() {
```

Comment:
The "persistence" you're testing is only that the # of stars stays the same when the rating bar is hidden and then shown again, right?  It would be more interesting to test that the number of starts persists across launches of the app.